### PR TITLE
Added a basic static site

### PR DESCRIPTION
**Pull request by rishabluthra**
I made a Gatsby documentation site with all the information from the markdown pages, I also configured the links to be used in GitHub pages.

I had to edit the markdown a bit to fit Gatsby's formatting.

### CODE REVIEW

File: shoebox-site/config.toml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+siteTitle="Shoebox"
+baseColor = "#FF3D2E"
+linkPrefix = "/shoebox"
+docPages = [
```
Comment (distalx) [inline shoebox-site/config.toml]: can you please replace `docs` with `topics` ?

File: shoebox-site/pages/_template.jsx [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+                    textDecoration: 'none',
+                    paddingLeft: rhythm(1/2),
+                    paddingRight: rhythm(1/2),
+                    paddingBottom: rhythm(3/4),
+                    marginBottom: rhythm(-1),
+                    paddingTop: rhythm(1),
+                    marginTop: rhythm(-1),
+                  }}
+                >
+                  Collection
```
Comment (distalx) [inline shoebox-site/pages/_template.jsx]: can you please replace `Collection` with `Topics` ?

**Review by distalx — COMMENTED** [review]
@SuparSquid  
This is great work, thanks so much! Just requires a few minor changes  :)

**Comment by rishabluthra** [comment]
@distalx 
Hey, I committed your changes.

**Comment by distalx** [comment]
@SuparSquid Thanks. 🚀

**Pull request merged**